### DISCUSSION TITLE
Merge Commit and PR result ?

### DISCUSSION THREAD

**Discussion by fletort** [testspace-com/support#35] [Ideas]
(github action context)
In my worflow my job run twice: on the commit on the branch, and also on the related PR opened (so on teh same commit).
One additional test is made in the PR context.
Often the "branch" worflow ends after the "PR worflow"... So the testsapce current view for my branch so the result with less tests

<img width="478" alt="Capture d’écran 2024-07-15 162116" src="https://github.com/user-attachments/assets/31d65981-ba5b-4147-bd70-b4c0f30f3955">

It can be possible to merge theses results ?

**Comment by munderseth** [comment]
Hi @fletort, currently at this time will do not have the functionality to merge the two different results.
